website/pages/docs/drivers/external/containerd.mdx

```diff
@@ -123,6 +126,27 @@ config {
 }
 ```
 
+- `host_dns` - (Optional) `true` (default) or `false` By default, a container
+  launched using `containerd-driver` will use host `/etc/resolv.conf`. This is
+  similar to [Docker's behavior]. However, if you don't want to use
+  host DNS, you can turn off this flag by setting `host_dns=false`.
+
+- `seccomp` - (Optional) Enable default seccomp profile. List of [`allowed syscalls`][].
```

Comment:
Sorry, looks like we missed a spot: https://nomad-i2tmn1oma.vercel.app/docs/drivers/external/containerd#seccomp

```suggestion
- `seccomp` - (Optional) Enable default seccomp profile. List of [allowed syscalls].
```

Reply:
Fixed.

Reply:
Thanks!